src/components/task.vue

```diff
@@ -365,6 +372,9 @@ export default {
         // There could be another task in the parent, so don't emit completed
         return;
       }
+      if (this.parentRequest) {
+        this.$emit('completed', this.parentRequest);
+      }
       this.$emit('completed', this.requestId);
```

Comment:
the completed event with the requestId always will be sent  and I don't know if it can have side effects. It seems that it will be better to put this line inside an "else" block of the above "if" condition, so

Reply:
If there is a parentRequest and task allows interstitial will return and not emit